doc/source/serve/deployment-graph/chain_nodes_same_class_different_args.md

```diff
@@ -46,8 +27,8 @@ class Model:
       return input +  self.weight
```

Comment:
for line above, just using `ray.init()` will be sufficient, by default we use num_cpus = # of cores on machine